collector/pom.xml

```diff
@@ -29,7 +29,8 @@
     <dependency>
       <groupId>org.yaml</groupId>
       <artifactId>snakeyaml</artifactId>
-      <version>1.26</version>
+      <!-- uses an older version because 2.24+ requires Java 7 -->
+      <version>1.16</version>
```

Comment:
Can we use .23?

Reply:
In theory, yes, but I don't know if there will be any gotchas.
I'm stuck with some last century systems, and satisfied with 1.16 so far. But I can try 1.23 later this week.

Reply:
.18 is what the previous bump was trying to avoid, so let's see if we can make everyone happy.